global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "2.0.0-preview2-006215"
```

Comment:
Pinning the cli and packages version to have this build successfully for rel/15.3 branch.